syntax="proto3";

enum RequestCode
{
  UserLogin=0;
  Register=1;
  AesFenfa=2;
};

enum RespondCode
{
  LoginOK=0;
  RegisterOK=1;
  RsaFenFa=2;
  AesVerifyOK=3;
  Failed=4;
};


message Information
{
  string userName=1;
  //预留三个空位
  string data1=2;
  string data2=3;
  string data3=4;

  RequestCode reqcode=5;
  RespondCode rescode=6;
}
